sync/optimistic.md

```diff
@@ -99,14 +106,21 @@ behaviours without regard for optimistic sync.
 ### When to optimistically import blocks
 
 A block MAY be optimistically imported when
-`is_optimistic_candidate_block(opt_store, current_slot, block)` returns
-`True`. This ensures that blocks are only optimistically imported if either:
+`is_optimistic_candidate_block(opt_store, current_slot, block)` returns `True`.
+This ensures that blocks are only optimistically imported if one or more of the
+following are true:
 
+1. The parent of the block has execution enabled.
 1. The justified checkpoint has execution enabled.
 1. The current slot (as per the system clock) is at least
    `SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY` ahead of the slot of the block being
    imported.
 
+In effect, there are restrictions on when a *merge block* can be optimistically
```

Comment:
I would be consistent denoting merge transition block. As it's being called "merge block" and "transition block" in two sister sentences is a bit confusing.

Reply:
Good point. I noticed we used "merge block" in other parts of the document so I standardized on that.